README.md

```diff
@@ -82,7 +96,7 @@ The root module serves as the entry point for one-app to load an application.
           | ------------------------------- |
 ```
 
-It is possible for your application to consist of only the root module, however most application will want to take advantage of code splitting using [Holocron](https://github.com/americanexpress/holocron) and have the root module load other modules.
+It is possible for your application to consist of only the root module, however most application will want to take advantage of code splitting using [Holocron](https://github.com/americanexpress/holocron) and have the root module load other modules. More on this in [routing](#-routing).
 
 For a module to act as the root module the only requirements are:
```

Comment:
I would add a `** 📘 More Information **` section at the end of this tutorial with any all topics you touched on that apply to the API docs. Example: https://github.com/americanexpress/one-app#loadmoduledata

Reply:
added 5d97705